cmd/remove.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/wtsi-hgi/go-softpack-builder/remove"
+	"github.com/wtsi-hgi/go-softpack-builder/s3"
+)
+
+var removeCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove an environment",
+	Long: `Remove an environment.
+
+Remove an existing environment; it's entry in the Core artefacts repo, the
+module files and the singularity image and symlinks.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			die("require a environment path")
+		}
+
+		const maxArgs = 2
+
+		if len(args) < maxArgs {
+			die("require a environment version")
+		}
+
+		if len(args) != maxArgs {
+			die("unexpected arguments")
+		}
+
+		conf := getConfig()
+
+		s, err := s3.New(conf.S3.BuildBase)
+		if err != nil {
+			die(err.Error())
+		}
+
+		envPath := filepath.Clean(string(filepath.Separator) + args[0])[1:]
```

Comment:
Having this as a well-named private function with some docs would be helpful to understand what's going on here and why it's needed.
If it's the same thing as done in the remove package, make it a public method of that package.